### PR TITLE
fix(ios): fix duplicate reference warnings

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/TIMOB-27376

Fixes the following warnings that occur for `every build`:
```
>> [ERROR] 2019-08-22 23:51:47.821 xcodebuild[39858:743045] warning:  The file reference for "" is a member of multiple groups ("TableView & ListView" and "UI"); this indicates a malformed project.  Only the membership in one of the groups will be preserved (but membership in targets will be unaffected).  If you want a reference to the same file in more than one group, please add another reference to the same path.
>> [ERROR] 2019-08-22 23:51:47.821 xcodebuild[39858:743045] warning:  The file reference for "" is a member of multiple groups ("UI" and "UI"); this indicates a malformed project.  Only the membership in one of the groups will be preserved (but membership in targets will be unaffected).  If you want a reference to the same file in more than one group, please add another reference to the same path.
>> [ERROR] 2019-08-22 23:51:47.821 xcodebuild[39858:743045] warning:  The file reference for "" is a member of multiple groups ("UI" and "UI"); this indicates a malformed project.  Only the membership in one of the groups will be preserved (but membership in targets will be unaffected).  If you want a reference to the same file in more than one group, please add another reference to the same path.
>> [ERROR] 2019-08-22 23:51:47.821 xcodebuild[39858:743045] warning:  The file reference for "" is a member of multiple groups ("UI" and "UI"); this indicates a malformed project.  Only the membership in one of the groups will be preserved (but membership in targets will be unaffected).  If you want a reference to the same file in more than one group, please add another reference to the same path.
```

### CODE REVIEW

File: iphone/iphone/Titanium.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1068,7 +1068,8 @@
 				24ADC5081299F4730014DB75 /* Clipboard */,
 				24CA8C5A111165420084E2DE /* ImageView */,
 				24D06FDE11D020E600F615D8 /* iOS */,
-				502A68CF16EEAA4A00661EF6 /* ListView */,
+				DBB0FE8D20D9364E00B85333 /* Matrix 2D/3D */,
```
Comment (hansemannn) [inline iphone/iphone/Titanium.xcodeproj/project.pbxproj]: Note that this one was completely missing.

**Review by janvennemann — APPROVED** [review]
LGTM!

**Comment by lokeshchdhry** [comment]
Build fails with:
```
[ERROR] :  ** BUILD FAILED **
[ERROR] :  The following build commands failed:
[ERROR] :  	CompileC /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/build/Intermediates/z123.build/Debug-iphonesimulator/z123.build/Objects-normal/x86_64/TiUIListSectionProxy.o /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/TiUIListSectionProxy.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
[ERROR] :  	CompileC /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/build/Intermediates/z123.build/Debug-iphonesimulator/z123.build/Objects-normal/x86_64/TiUIListView.o /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/TiUIListView.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
[ERROR] :  	CompileC /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/build/Intermediates/z123.build/Debug-iphonesimulator/z123.build/Objects-normal/x86_64/TiUIListViewProxy.o /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/TiUIListViewProxy.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
[ERROR] :  	CompileC /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/build/Intermediates/z123.build/Debug-iphonesimulator/z123.build/Objects-normal/x86_64/TiUIListItemProxy.o /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/TiUIListItemProxy.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
[ERROR] :  	CompileC /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/build/Intermediates/z123.build/Debug-iphonesimulator/z123.build/Objects-normal/x86_64/TiUIListItem.o /Users/lchoudhary/Desktop/workspaces/workspace_2019/z123/build/iphone/TiUIListItem.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
[ERROR] :  (5 failures)
```

**Comment by hansemannn** [comment]
The project file free needed a re-referencing of the TiUIList*** classes, so now it should work @lokeshchdhry .

**Comment by lokeshchdhry** [comment]
FR Passed.

No warnings/errors seen.

Studio Ver:       5.1.3.201907112159
SDK Ver:          8.2.0 local build
OS Ver:           10.14.5
Xcode Ver:        Xcode 11.0
Appc NPM:         4.2.15-1
Appc CLI:         7.1.1-master.6
Daemon Ver:       1.1.3
Ti CLI Ver:       5.2.1
Alloy Ver:        1.14.1
Node Ver:         10.16.1
NPM Ver:          6.9.0
Java Ver:         10.0.2
Simulator: Iphone ios 13

**Pull request merged**